### PR TITLE
EMMO prefix and imports fixed in computation.ttl --> 1.0.0-RC1

### PR DESCRIPTION
**Pull request by hothello**
I have changed the prefix of EMMO URIs to a standard `emmo:EMMO_hex_hex_hex_hex_hex` format and redirected the imports to the beta7.
Note to Jesper: the redirect to `https://w3id.org/emmo/multiperspective/persholistic` is not resolved, while that to `https://w3id.org/emmo/1.0.0-rc1/multiperspective/persholistic` is.

### CODE REVIEW

**Comment by hothello** [comment]
The idea behind the domain ontology `openmodel.ttl` is to have a taxonomy collecting the materials models, relations, and software used to demonstrate the various SSs. The selected classes in the inferred ontology follow:
image

File: computation.ttl [hunk 1]
```diff
@@ -3,15 +3,17 @@
...
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @base <http://emmo.info/computation#> .
 
 <http://emmo.info/computation> rdf:type owl:Ontology ;
                                 owl:versionIRI <http://emmo.info/computation/0.1> ;
-                                owl:imports <http://emmo.info/emmo/computer-science/multiperspective/persholistic> ,
-                                            <https://raw.githubusercontent.com/emmo-repo/EMMO/computer-science/disciplines/computerscience.ttl> ;
+                                owl:imports <https://w3id.org/emmo/1.0.0-beta7/disciplines/computerscience> ,
+                                            <https://w3id.org/emmo/1.0.0-beta7/multiperspective/persholistic> ,
+                                            <https://w3id.org/emmo/1.0.0-beta7/perspectives/semiotics> ;
```
Comment (francescalb) [inline computation.ttl:16]: According to the catalog we are not supposed to use beta-7, but 
https://emmo-repo.github.io/versions/1.0.0-rc1/emmo.ttl.

I see that the catalog also should be updated.

File: computation.ttl [hunk 2]
```diff
@@ -39,161 +41,245 @@ owl:qualifiedCardinality rdf:type owl:AnnotationProperty .
...
+                                           skos:prefLabel "MPIProcessorsPerNode"@en ;
+                                           emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "Number of processors that can be used by MPI on a compute node."@en .
+
+
+###  http://emmo.info/computation#EMMO_305f7d77_6e06_496d_90a6_95db6a3a9166
+:EMMO_305f7d77_6e06_496d_90a6_95db6a3a9166 rdf:type owl:Class ;
+                                           rdfs:subClassOf emmo:EMMO_13fea749_0b3b_4756_9c81_22cce620fc25 ,
+                                                           emmo:EMMO_af7b627d_6528_4fdb_97df_ba4579c4bb5d ;
+                                           skos:prefLabel "XML"@en ;
+                                           emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "Extensible Markup Language (XML) is a markup language and file format for storing, transmitting, and reconstructing arbitrary data. It defines a set of rules for encoding documents in a format that is both human-readable and machine-readable."@en .
```
Comment (francescalb) [inline computation.ttl:54]: I do not think that we should have file formats in the computation ontology. I prefer having them all in a separate ontology if they are needed.

File: openmodel.ttl [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+@prefix emmo: <https://w3id.org/emmo#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@base <https://w3id.org/emmo/domain/openmodel#> .
+
+<https://w3id.org/emmo/domain/openmodel> rdf:type owl:Ontology ;
+                                          owl:versionIRI <https://w3id.org/emmo/domain/openmodel> ;
+                                          owl:imports <https://w3id.org/emmo/1.0.0-beta7/disciplines/chemistry> ,
```
Comment (francescalb) [inline openmodel.ttl:15]: let's use rc-1 as stated in the catalog.

File: openmodel.ttl
Comment (francescalb) [inline openmodel.ttl:1]: What is the purpose of this ontology? What should it be used for? It seems to me that it is mixing concepts from different domains, e.d. atomistic simulations concepts on a more general basis and specific AiiDA-relevant commands. What are we going to use this for in OpenModel?

File: computation.ttl (hunk 1 above)
Comment (hothello) [reply computation.ttl:16]: fixed

File: computation.ttl (hunk 2 above)
Comment (hothello) [reply computation.ttl]: File formats removed. I will ask the experts @pierluigidelnostro and Daniele!

File: openmodel.ttl
Comment (hothello) [reply openmodel.ttl:1]: This ontology has now been removed. Is there anything else to check?

**Review by francescalb — APPROVED** [review]
Looks fine, but be careful, seems like you are 29 commits behing master.

**Comment by hothello** [comment]
All good, except that I have accidentally removed the class BASH from `computation.ttl`. I will reinstate it on the branch `ss1_dev` and it will be integrated back on the next PR.

**Pull request merged**